jsonloader.js

```diff
@@ -0,0 +1,27 @@
+loadJSON = (callback) => {
+
+    var xobj = new XMLHttpRequest();
+    xobj.overrideMimeType("application/json");
+    xobj.open('GET', './_data/orgs.json', true);
```

Comment:
this file wasnt being published, but I've fixed that: https://github.com/summerofcode/gsoc-orgs/commit/34243808e3d2afabd397c435756767bafdaf2d72